scripts/export-answers.js

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env node
 
 const BUCKET = "concordqa-report-data"
-const DIRECTORY = "partitioned-answers"
+const SOURCE = "authoring.staging.concord.org"
```

Comment:
I think our decision from earlier today is that this will look at all sources in Firestore. So it won't be necessary to keep track of this SOURCE.

Reply:
Yes, I am looking into changing this now.

Reply:
This is now updated.

Reply:
So this SOURCE variable is just in the export-answers script and a developer will ned to set this appropriately before running the script.  We should make this a parameter, but that could be in a different PR which is focused on the export-answers script.

Reply:
Yes, I'm sorry, I missed the context of which script we were referring to when replying. In export-answers this is currently per-source. It could have been made into a parameter forcing a dev to specify it explicitly when running the script, or we could have made it work across all sources (though I think per-source makes sense when running the entire database dump), but it was not handled as part of this PR.